core/src/test/java/test/org/apache/spark/JavaAPISuite.java

```diff
@@ -960,7 +960,7 @@ public void textFiles() throws IOException {
     rdd.saveAsTextFile(outputDir);
     // Read the plain text file and check it's OK
     File outputFile = new File(outputDir, "part-00000");
-    String content = Files.asCharSource(outputFile, StandardCharsets.UTF_8).read();
+    String content = java.nio.file.Files.readString(outputFile.toPath());
```

Comment:
`java.nio.file.Files.readString` will be replaced to `Files.readString` when we finish the migration in this file.